app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -657,6 +666,15 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             binding.blurView.setBlurEnabled(false)
         }
 
+        // Enable pinch to zoom
+        if (presenter.isPinchToZoomEnabled()) {
+            webView.getSettings().setBuiltInZoomControls(true)
+            webView.getSettings().setDisplayZoomControls(false)
```

Comment:
Because this is `false` for both settings, perhaps it would be better to move this to the setup block (inside `webView.apply {}` with the other settings)

Reply:
OK. removed redundant code